traffic_monitor/tmcheck/tmcheck.go

```diff
@@ -44,7 +43,7 @@ const TrafficMonitorStatsPath = "/publish/Stats"
 
 func getClient() *http.Client {
 	return &http.Client{
-		Transport: &http.Transport{TLSClientConfig: &tls.Config{InsecureSkipVerify: true}},
+		Transport: &http.Transport{},
```

Comment:
I believe the `Transport` property doesn't need to be set at all if you're not doing anything special with it.